dataplanes/vpp/conf/startup.conf

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
Pedantic but important nit: In the interest of making this easier to understand for new contributors, we may want to put this in the directory `dataplanes/vpp/conf/vpp/startup.conf` so it's obvious to new contributors this is a VPP startup configuration file.